optuna/visualization.py

```diff
@@ -92,6 +99,164 @@ def _get_intermediate_plot(study):
     return figure
 
 
+def plot_contour(study, params=[]):
+    # type: (Study, List[str]) -> None
+    """Plot the parameter relationship as contour plot in a study.
+
+        Note that, If a parameter contains missing values, a trial with missing values is not
+        plotted.
+
+    Example:
+
+        The following code snippet shows how to plot the parameter relationship as contour plot
+        inside Jupyter Notebook.
```

Comment:
Related to https://github.com/pfnet/optuna/pull/513#discussion_r326118285, I think that "inside Jupyter Notebook" can be removed.